python/cudf/cudf/core/index.py

```diff
@@ -779,18 +779,44 @@ def __len__(self):
         return len(self._values)
 
     def __repr__(self):
-        vals = [self._values[i] for i in range(min(len(self), 10))]
-        return (
-            "{}({}, dtype={}".format(
-                self.__class__.__name__, vals, self._values.dtype
-            )
-            + (
-                ", name='{}'".format(self.name)
-                if self.name is not None
-                else ""
+        from pandas._config import get_option
+
+        max_seq_items = get_option("max_seq_items") or len(self)
```

Comment:
What happens if this option is set to 0?

Reply:
It will print all the values and there will not be any truncation. It is same with pandas as well.